src/style.css

```diff
@@ -1 +1,31 @@
 /* add styles here */
+* {
+  margin: 0;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser. Try to add styles to the specific classes that need them.